valhalla/actor.py

```diff
@@ -79,6 +79,10 @@ def config_path(self):
     def route(self, req: Union[str, dict]):
         return super().route(req)
 
+    @dict_or_str
+    def optimized_route(self, req: Union[str, dict]):
+        return super().route(req)
```

Comment:
argh, I just realize this is not right: we need to call `super().optimized_route(req)`.. right now you're just getting an un-optimized route